cypress/e2e/setup-scripts/seed_contributor.py

```diff
@@ -0,0 +1,19 @@
+"""DB seed script for e2e cypress contributor tests."""
+
+from django.contrib.auth.models import User
+from projects.models import Project
+
+
+USERNAME = "e2e_tests_contributor_tester"
+EMAIL = "no-reply-contributor@scilifelab.se"
+PWD = "test12345"
+
+
+# Create the contributor user
+user = User.objects.create_user(USERNAME, EMAIL, PWD)
+user.save()
+
+# Create a dummy project to be deleted by the contributor user
+Project.objects.create_project(
+    name="e2e-delete-proj-test", owner=user, description="", repository=""
+)
```

Comment:
I'll ask, just in case

Don't you have to call `.save()` on this?

Reply:
It is not needed because create_project executes create() which instantiates the object and persists to the db in one step.